ebusd-2.1.x/de/vaillant/15.uih.csv

```diff
@@ -48,7 +48,9 @@ r;wi,,EhpHeatBufferAvailable,EhpHeatBufferAvailable,,,,"5C00",,,onoff,,,Puffersp
 [SW>=250]r,,SysWithCoolingSupport,Kühlsystem erkannt,,,,"9B00",,,onoff,,,
 [SW>=362]r,,reserve1,reserve 1,,,,"9D00",,,UCH,,,for future use
 # ##### HK Namen und Tel. Nummer #####,,,,,,,,,,,,,
+r;w,,HcName1,Name HK1,,,,"4100",,,hcname,,,Name für HK1
```

Comment:
is there really a heating circuit 1? usually this is some internal stuff and I doubt that the name for this is anywhere visible on the controller

Reply:
Hello John, yes there is - see results from the reads of the corresponding calls:
read -c uih HcName1
**HK1**       
read -c uih HcName2
**HK2**       
read -c uih HcName3
**HK3**       
read -c uih HcName4
**HK4**       
read -c uih HcName16
**Speicher**